.github/bin/download-maven-dependencies.sh

```diff
@@ -13,7 +13,7 @@ $RETRY $MAVEN_ONLINE -B de.qaware.maven:go-offline-maven-plugin:resolve-dependen
 
 # Enable common profiles to make sure their plugin dependencies are downloaded as well
 # GIB should be disabled even though it's profile is active, to make sure it doesn't skip any submodules
-$RETRY $MAVEN_ONLINE -B -P ci,errorprone-compiler ${MAVEN_GIB} -Dgib.disable dependency:go-offline
+$RETRY $MAVEN_ONLINE -B -P ci,errorprone-compiler ${MAVEN_GIB} -Dgib.disable dependency:go-offline -Dsilent
 $RETRY $MAVEN_ONLINE -B -P ci,errorprone-compiler ${MAVEN_GIB} -Dgib.disable de.qaware.maven:go-offline-maven-plugin:resolve-dependencies
```

Comment:
what about this?

Reply:
and also the separate invocation for `pinot`?

Reply:
These don't produce excessive logs, see here: https://github.com/trinodb/trino/actions/runs/4391516912/jobs/7690724648

Reply:
got it, looks good then